### PR TITLE
fix(modal): limit setting modal drag range

### PR DESCRIPTION
**Pull request by Soya-xy**
Before:
https://github.com/user-attachments/assets/1fcda12c-b3ce-491a-b35e-644ceb95aa65

After:

https://github.com/user-attachments/assets/1763451f-94ad-4a53-9965-2487373811fa

### CODE REVIEW

**Comment by Soya-xy** [comment]
fix: #577

File: apps/renderer/src/components/ui/modal/stacked/modal.tsx [hunk 1]
```diff
@@ -340,6 +340,12 @@ export const ModalInternal = memo(
                   dragListener={false}
                   dragMomentum={false}
                   dragConstraints={edgeElementRef}
+                  onMeasureDragConstraints={(constraints) => {
+                    return {
+                      ...constraints,
+                      top: constraints.top + 32,
```
Comment (Innei) [inline apps/renderer/src/components/ui/modal/stacked/modal.tsx:346]: Only plus on Windows

File: apps/renderer/src/components/ui/modal/stacked/modal.tsx [hunk 2]
```diff
@@ -340,6 +340,12 @@ export const ModalInternal = memo(
                   dragListener={false}
                   dragMomentum={false}
                   dragConstraints={edgeElementRef}
+                  onMeasureDragConstraints={(constraints) => {
```
Comment (Innei) [inline apps/renderer/src/components/ui/modal/stacked/modal.tsx:343]: Memo this function

**Review by Innei — CHANGES_REQUESTED** [review]

**Review by Innei — APPROVED** [review]
Nice job

**Pull request merged**